components/profile/header/searching-for-club-modal.tsx

```diff
@@ -14,54 +15,61 @@ const SearchingForClubModal = ({
   status: { searchingForClub },
   open,
   closeModal,
-}: IProps) => (
-  <Modal
-    title="Szukam klubu"
-    show={open}
-    closeModal={closeModal}
-    btnSecondaryIcon={
-      <StarIcon style={{ height: '1.5rem', width: '1.5rem' }} />
-    }
-    btnPrimaryIcon={<SendIcon />}
-    btnPrimaryText="Napisz"
-  >
-    <div className="flex flex-col gap-7 pb-8 font-inter">
-      <LocalizationBanner
-        localizationText={searchingForClub?.localization.name || ''}
-        range={searchingForClub?.localization.range || 0}
-      />
+}: IProps) => {
+  useEffect(() => {
+    if (open) document.body.style.overflow = 'hidden'
+    else document.body.style.overflow = 'unset'
+  }, [open])
```

Comment:
ios fix and maybe we can move it to Modal component itself ?
```suggestion
  useEffect(() => {
 if (open) {
      document.body.style.overflow = 'hidden'
      document.body.style.touchAction = 'none'
    } else {
      document.body.style.overflow = 'unset'
      document.body.style.touchAction = 'auto'
    }
  }, [open])
```

Reply:
nice 👍